src/main/scala/io/iohk/ethereum/validators/BlockHeaderValidator.scala

```diff
@@ -18,6 +21,14 @@ object BlockHeaderValidatorImpl {
   val MaxPowCaches: Int = 2 // maximum number of epochs for which PoW cache is stored in memory
 
   class PowCacheData(val cache: Array[Int], val dagSize: Long)
+
+  // FIXME [EC-331]: this is used to speed up ETS Blockchain tests. All blocks in those tests have low numbers (1, 2, 3 ...)
+  // so keeping the cache for epoch 0 avoids recalculating it for each individual test. The difference in test runtime
+  // can be dramatic - full suite: 26 hours vs 21 minutes on same machine
+  // It might be desirable to find a better solution for this - one that doesn't keep this cache unnecessarily
+  lazy val epoch0PowCache = new PowCacheData(
+    cache = Ethash.makeCache(0),
+    dagSize = Ethash.dagSize(0))
 }
```

Comment:
The downside of keeping this cache is unnecessary use of disk space? How much?

Reply:
imo it won't be too hard to be able to configure `BlockHeaderValidator` with a `PowCache` and use one that keeps epoch0 for tests

Reply:
Not disk space - RAM, see [EC-331](https://iohk.myjetbrains.com/youtrack/issue/EC-331) for details